neuraxle/base.py

```diff
@@ -948,3 +1006,83 @@ class ResumableStepMixin:
     @abstractmethod
     def should_resume(self, data_container: DataContainer) -> bool:
         raise NotImplementedError()
+
+
+class Barrier(NonFittableMixin, NonTransformableMixin, BaseStep, ABC):
```

Comment:
`NonFittableMixin, NonTransformableMixin` or `NonTransformableMixin, NonFittableMixin`? Review order of the overriding of `handle_fit_transform` method.

Reply:
doesn't matter , but removed it

Reply:
why removing? it seems fine like currently with 4 inheriting classes then.